### PR TITLE
feat(send to): Add option to clear the publish schedule date&time

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: scripts/apps/authoring/views/send-item.html [hunk 1]
```diff
@@ -52,7 +52,9 @@
         </div>
 
         <div sd-toggle-box id="publishScheduleTimestamp" ng-if="mode === 'authoring' && showPublishSchedule()" data-title="{{ 'Publishing Schedule' | translate }}" data-open="true" data-style="dark">
-            <label class="label--lite" translate>Publish Schedule</label>
+            <label class="label--lite" translate>Publish Schedule
+                <button type="button" class="btn" ng-click="item.publish_schedule_date=null; item.publish_schedule_time=null;" title="Clear Publish Schedule" sd-hotkey="ctrl+shift+c"><i class="icon-remove-sign icon-white"></i></button>
```
Comment (mdhaman) [inline scripts/apps/authoring/views/send-item.html]: `ctrl+shift+c` is shortcut for developer tools.

File: scripts/apps/authoring/views/send-item.html [hunk 2]
```diff
@@ -52,7 +52,9 @@
         </div>
 
         <div sd-toggle-box id="publishScheduleTimestamp" ng-if="mode === 'authoring' && showPublishSchedule()" data-title="{{ 'Publishing Schedule' | translate }}" data-open="true" data-style="dark">
-            <label class="label--lite" translate>Publish Schedule</label>
+            <label class="label--lite" translate>Publish Schedule
```
Comment (petrjasek) [inline scripts/apps/authoring/views/send-item.html]: now this button would be part of translate string and its title is not translated

Comment (ioanpocol) [inline scripts/apps/authoring/views/send-item.html]: fixed

Comment (petrjasek) [inline scripts/apps/authoring/views/send-item.html]: it should be like `<label class="label--lite">{{ :: 'Publish Schedule' | translate }}`

File: scripts/apps/authoring/views/send-item.html [hunk 3]
```diff
@@ -52,7 +52,9 @@
...
+                <button type="button" class="btn" ng-click="item.publish_schedule_date=null; item.publish_schedule_time=null;" title="{{ 'Clear Publish Schedule' | translate" sd-hotkey="ctrl+shift+x"><i class="icon-remove-sign icon-white"></i></button>
```
Comment (petrjasek) [inline scripts/apps/authoring/views/send-item.html]: and here translate is missing `}}`

**Pull request merged**